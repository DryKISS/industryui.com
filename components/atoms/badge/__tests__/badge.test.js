import React from 'react'
import { Badge } from '../'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer
    .create(<Badge>Majid</Badge>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
