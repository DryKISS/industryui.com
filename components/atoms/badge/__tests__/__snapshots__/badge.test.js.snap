// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="sc-bdVaJa cxHFmo"
  itemProp="keywords"
/>
`;
